src/Dfe.PlanTech.AzureFunctions/Mappings/RecommendationChunkUpdater.cs

```diff
@@ -29,17 +29,16 @@ public override MappedEntity UpdateEntityConcrete(MappedEntity entity)
     private void RemoveOldAssociatedChunkContents(RecommendationChunkDbEntity existingRecommendationChunk)
     {
         var contentsToRemove = Db.RecommendationChunkContents
```

Comment:
Remove unused var

Reply:
..or, ignore further comments, and reassign to var and have closing  Db.RecommendationChunkAnswers.RemoveRange(contentsToRemove);

Reply:
Arr good spot. Changed it for testing then completely forgot to change it back. Sorted 👍